pkg/controller/kieapp/defaults/defaults.go

```diff
@@ -113,9 +113,44 @@ func GetEnvironment(cr *api.KieApp, service kubernetes.PlatformService) (api.Env
 	if err != nil {
 		return api.Environment{}, err
 	}
+	setProductLabels(cr, &mergedEnv)
 	return mergedEnv, nil
 }
 
+func setProductLabels(cr *api.KieApp, env *api.Environment) {
+	consoleComponent := "Console"
```

Comment:
Does it stands for Business Central?
maybe, isn't better call it as business central instead?

Reply:
Note that it could be DC as well

Reply:
yeah, good point.

Reply:
maybe... just below this we do a lookup for `FriendlyName` from these vars but wanted a default string in case env isn't matched -
https://github.com/kiegroup/kie-cloud-operator/blob/2fdcae9c54cc1d89c43b47adabc41a3c34561091/pkg/controller/kieapp/constants/constants.go#L262-L264

Reply:
i had to switch to all lower-case and hyphenated strings for the label strings.

Reply:
with this change... kie-servers would be labeled like so...
```yaml
      labels:
        com.redhat.product-name: process-automation
        app: rhpam-authoring-ha
        com.redhat.component-type: application
        com.redhat.product-version: 7.8.0
        deploymentConfig: rhpam-authoring-ha-kieserver
        application: rhpam-authoring-ha
        com.redhat.component-version: 7.8.0
        services.server.kie.org/kie-server-id: rhpam-authoring-ha-kieserver
        com.redhat.component-name: kie-server
        service: rhpam-authoring-ha-kieserver
```
console like...
```yaml
      labels:
        com.redhat.product-name: process-automation
        app: rhpam-authoring-ha
        com.redhat.component-type: application
        com.redhat.product-version: 7.8.0
        deploymentConfig: rhpam-authoring-ha-rhpamcentr
        application: rhpam-authoring-ha
        com.redhat.component-version: 7.8.0
        com.redhat.component-name: businesscentral
        service: rhpam-authoring-ha-rhpamcentr
```

Reply:
@tchughesiv We don't really need to distinguish Business Central and Decision Central. Some time ago we made that switch and we don't even mention Decision Central in the docs anymore: https://access.redhat.com/documentation/en-us/red_hat_decision_manager/7.7/

Reply:
ok @ibek ... how would you like to name console components then?

Reply:
business-central ... the same name is used across the community and products.